samples/junit-tests/src/test/java/com/yelp/codegen/generatecodesamples/FormatEndpointTest.kt

```diff
@@ -0,0 +1,92 @@
+package com.yelp.codegen.generatecodesamples
+
+import com.yelp.codegen.generatecodesamples.apis.ResourceApi
+import com.yelp.codegen.generatecodesamples.models.FormatResponses
+import com.yelp.codegen.generatecodesamples.tools.MockServerApiRule
+import okhttp3.mockwebserver.MockResponse
+import org.junit.Assert.assertEquals
+import org.junit.Rule
+import org.junit.Test
+import org.threeten.bp.LocalDate
+import org.threeten.bp.ZonedDateTime
+import org.threeten.bp.ZoneId
+
+class FormatEndpointTest {
+
+    @get:Rule
+    val rule = MockServerApiRule()
+
+    @Test
+    fun formatEndpoint_withEnumFormat() {
+        rule.server.enqueue(MockResponse().setBody("""
+            {
+                "enum_property": "VALUE1"
+            }
+        """.trimIndent()))
```

Comment:
nit. is this needed? I think that this is not needed and eventually I would suggest to remove it as it does not add any value to the test

Reply:
`is this needed?` -> What are you referring to?
The MockResponse here is necessary as otherwise the test will fail.

The `.trimIndent()` is automatically added by IntelliJ when opening a multiline string and makes sure the strings are well trimmed. I can eventually remove all of them but I don't think they hurt so much.

Reply:
I wasn't aware about the intellij smart additions